templates/layout.hbs

```diff
@@ -5,12 +5,18 @@
     <meta charset="utf-8" />
     <link rel="stylesheet" href="/_hal-browser/assets/themes/lfo/main.css" type="text/css" />
     <link rel="stylesheet" href="/assets/form.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;600&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Unfortunately, since this is a security product, I don't think we should reach out to other services.

We could embed our own fonts if we want to, but google fonts tracks users and I think this sends the wrong message.